Movie/Movie/CinemaListScreen/CinemaListViewController.swift

```diff
@@ -1,11 +1,255 @@
-// ViewController.swift
+// CinemaListViewController.swift
 // Copyright © RoadMap. All rights reserved.
 
 import UIKit
 
-/// Главный экран
-class ViewController: UIViewController {
+/// Экран со списком кинофильмов.
+final class CinemaListViewController: UIViewController {
+    // MARK: - Private types
+
+    enum CellIdentifiers: String {
+        case cinemaList = "CinemaListCell"
+    }
+
+    // MARK: - Private properties
+
+    private var model: CinemaInfoProtocol? {
+        willSet {
+            DispatchQueue.main.async {
+                self.descriptionScreenHelperArray = newValue?.results.map { result in
+                    var imageData = Data()
+                    let condition = NSCondition()
+
+                    NetworkManager.manager.getImage(
```

Comment:
```suggestion
                    NetworkManager.manager.getImage(
```
запрос хотяб в метод вынеси)